src/tracker/tracker-game-source.vala

```diff
@@ -78,6 +70,7 @@ public class Games.TrackerGameSource : Object, GameSource {
 				continue;
 			}
 		}
+		query.foreach_game.begin(game_callback);
```

Comment:
We already are in an async method so let's chain up the asynchronous calls rather than starting another async method: `yield query.foreach_game (game_callback);`

Also, we put a space before param lists.